app/uk/gov/hmrc/ccn2connectivitytester/repositories/SoapMessageStatusRepository.scala

```diff
@@ -74,18 +74,18 @@ class SoapMessageStatusRepository @Inject() (mongoComponent: MongoComponent, app
     collection.withReadPreference(primaryPreferred())
       .findOneAndUpdate(
         filter = equal("messageId", Codecs.toBson(messageId)),
-        update = set("status", Codecs.toBson(newStatus.entryName)),
+        update = set("status", Codecs.toBson(newStatus.toString())),
```

Comment:
If the json formatter is in scope then you can loose the toString but I'm not sure which I prefer or which is consistent.